src/app/geolocation/geolocation.component.spec.ts

```diff
@@ -13,7 +14,10 @@ describe('GeolocationComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [ GeolocationComponent ],
-      imports:[RouterTestingModule, HttpClientModule,AgmCoreModule.forRoot({
+      imports:[RouterTestingModule, 
+        HttpClientModule,
+        SharedModule,
+        AgmCoreModule.forRoot({
         apiKey: 'AIzaSyCHC4hLZ_ogiMaGy9kVZl4u0gjocEyw3EA'
```

Comment:
I think this key belongs to you, not to the company, and I think we should have a note to change it later.